docs/devsetup/java/maven/index.md

```diff
@@ -106,7 +106,22 @@ work as they do normally in IntelliJ.
 
 [Eclipse's Maven plugin](https://eclipse.dev/m2e/)
 works reasonably well and can support the import of a multi-module Maven project.
-These instructions have not yet verified Eclipse's handling of the import of the ICU4J using a Maven build.
+
+1. It is highly recommended (but optional) to do the ICU work in a separate workspace. \
+   You can also import to a working-set. Whatever you prefer. \
+   A workspace allows you to configure the Eclipse options for ICU at workspace level,
+   not at module level. It also separates the (many) ICU modules from the other work
+   you do. The downside is that whenever you open Eclipse, you will have to select
+   the workspace to work on.
+   See the official Eclipse [Workspaces](https://www.ibm.com/docs/en/baw/23.0.x?topic=modules-workspaces) page for more.
+   1. From "Settings" -- "Startup and Shutdown" -- "Workspaces" check the
+   "Prompt for workspace on startup" option.
+   2. Restart Eclipse, and when prompted type a workspace name
+   (for example `/Users/john/workspace_icu`).
```

Comment:
Do these nested-list items format properly when their continuation lines are not further indented?

Reply:
They rendered fine in VS Code.
But I will indent them a bit more, just in case. Maybe GitHub renders differently.